src/plugins/bubble/bubble-fr.hbs

```diff
@@ -0,0 +1,64 @@
+---
+{
+	"title": "Bubble - Fonctionalité provisoire",
+	"language": "fr",
+	"category": "Plugiciels",
+	"description": "Example pratique d'utilisation du chat wizard",
+	"tag": "bubble",
+	"parentdir": "bubble",
+	"altLangPrefix": "bubble",
+	"dateModified": "2020-10-21"
+}
+---
+{{>alert-provisional-demos}}
+
+<div class="wb-prettify all-pre"></div>
+<div lang="en">
+<p><strong>Need translation</strong></p>
+
+<p>Below is the code to achieve the demo on this page.</p>
+<pre><code>&lt;button type="button" data-wb-doaction='{ "action": "openDialog", "source": "#mid-screen" }' class="wb-bubble trans-pulse"&gt;test&lt;/button&gt;
+&lt;section id="mid-screen" class="wb-overlay modal-content overlay-def wb-popup-mid"&gt;
+	&lt;header class="modal-header"&gt;
+		&lt;h2 class="modal-title"&gt;Middle screen overlay&lt;/h2&gt;
+	&lt;/header&gt;
+	&lt;div class="modal-body"&gt;
+		&lt;ol&gt;
+			&lt;li&gt;ordered list (&lt;code&gt;ol&lt;/code&gt;) first level default appearance&lt;/li&gt;
+			&lt;li&gt;ordered list (&lt;code&gt;ol&lt;/code&gt;) first level default appearance
+				&lt;ol&gt;
+					&lt;li&gt;ordered list (&lt;code&gt;ol&lt;/code&gt;) second level default appearance&lt;/li&gt;
+					&lt;li&gt;ordered list (&lt;code&gt;ol&lt;/code&gt;) second level default appearance
+						&lt;ol&gt;
+							&lt;li&gt;ordered list (&lt;code&gt;ol&lt;/code&gt;) third level default appearance&lt;/li&gt;
+							&lt;li&gt;ordered list (&lt;code&gt;ol&lt;/code&gt;) third level default appearance&lt;/li&gt;
+						&lt;/ol&gt;
+					&lt;/li&gt;
+				&lt;/ol&gt;
+			&lt;/li&gt;
+		&lt;/ol&gt;
+	&lt;/div&gt;
+&lt;/section&gt;</code></pre>
+
+<button type="button" data-wb-doaction='{ "action": "openDialog", "source": "#mid-screen" }' class="wb-bubble trans-pulse">test</button>
```

Comment:
Please remove the pulsing class named `trans-pulse`, since it should be added dynamically in Javascript.
```suggestion
<button type="button" data-wb-doaction='{ "action": "openDialog", "source": "#mid-screen" }' class="wb-bubble">test</button>
```
Same in the code sample above and in French too.